CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Changelog
 
+## TBD
+
+### Added
+
+- (delivery-expo) Explicitly mark failed payloads >1MB as not retryable [#65](https://github.com/bugsnag/bugsnag-expo/pull/65)
```

Comment:
was this one missing from a previous PR?

Reply:
yeah this one: https://github.com/bugsnag/bugsnag-expo/pull/65